### ISSUE TITLE
Add version metadata to windows `fluent-bit.exe` executable

### ISSUE THREAD

**Issue by jantari** [fluent/fluent-bit#4526]
**Is your feature request related to a problem? Please describe.**
None of the files in fluent-bit / td-agent-bit program directory have version metadata. Because of this, I cannot properly deploy td-agent-bit with ansible.

At least the main executable `./bin/fluent-bit.exe` should have a ProductVersion and FileVersion set.
This is required to be able to properly install and update td-agent-bit with ansibles [`win_package`](https://docs.ansible.com/ansible/latest/collections/ansible/windows/win_package_module.html) module because if I cannot specify a value for `creates_version` then the `win_package` module will not install a newer version of fluent-bit because there is already a version installed and ansible needs the file metadata to test the installed version and determine whether an update is needed.

**Describe the solution you'd like**
Add file metadata to windows build artifacts, specifically ProductVersion.

**Describe alternatives you've considered**
None

**Additional context**
File metadata is easy to view in the graphical file explorer or PowerShell:

```powershell
# just the main executable:
(Get-Item .\fluent-bit.exe).VersionInfo | Format-List
# or to test all files:
ls -Path 'C:\Program Files\td-agent-bit' -Recurse -File | Select -Expand VersionInfo
```

The first command currently shows the lack of metadata in 1.8.11:

```
OriginalFilename  :
FileDescription   :
ProductName       :
Comments          :
CompanyName       :
FileName          : C:\Program Files\td-agent-bit\bin\fluent-bit.exe
FileVersion       :
ProductVersion    :
IsDebug           : False
IsPatched         : False
IsPreRelease      : False
IsPrivateBuild    : False
IsSpecialBuild    : False
Language          :
LegalCopyright    :
LegalTrademarks   :
PrivateBuild      :
SpecialBuild      :
FileVersionRaw    : 0.0.0.0
ProductVersionRaw : 0.0.0.0
```

**Comment by jantari** [comment]
Hi,

I can see that he PR #4531 addressing this issue was merged on 27 Dec 2021, but I installed and tested td-agent-bit 1.8.12 today, which released on 28 Jan 2022 and should have the fix, but `fluent-bit.exe` still has empty metadata.

**Comment by patrick-stephens** [comment]
It was merged to master which is the 1.9 release rather than the dedicated 1.8 maintenance branch.
